core/src/impl/Kokkos_Error.hpp

```diff
@@ -113,9 +113,6 @@ class RawMemoryAllocationFailure : public std::bad_alloc {
   };
   enum class AllocationMechanism {
     StdMalloc,
-    PosixMemAlign,
-    PosixMMap,
-    IntelMMAlloc,
```

Comment:
I had commented on an individual (last?) commit.
No do not remove, deprecate them.

Reply:
it is not about me being happy or not, it is about not potentially breaking user code.